logging/zap/grpclogger.go

```diff
@@ -8,6 +8,8 @@ import (
 
 	"go.uber.org/zap"
 	"google.golang.org/grpc/grpclog"
+	"os"
+	"strconv"
 )
 
 // ReplaceGrpcLogger sets the given zap.Logger as a gRPC-level logger.
```

Comment:
Can we add a depreciation notice to this pointing at V2

Reply:
`// Deprecated: use ReplaceGrpcLoggerV2.`

Reply:
yep